spec/client_spec.js

```diff
@@ -419,15 +419,15 @@ describe('Client', function() {
         it('resolves when the request succeeds', function(done) {
           triggerEvent(subject, 'request:' + requestsCount + '.done', response);
           promise.then(function() {
-            expect(doneHandler).to.have.been.calledWith(response.responseArgs[0]);
```

Comment:
I'm actually very impressed that we've had this bug since feb 2015 and nobody has really bumped into it since now.